tests/bugs/test_bytecode.py

```diff
@@ -0,0 +1,52 @@
+"""
+A bytecode sequence that caused bug, calling to undefined opcode with filled stack
```

Comment:
I think the name of the file is too generic, `test_undefined_opcode.py` might be a bit more clear.

Reply:
We actually should have this one covered in undefined opcode tests. Where we fill up the stack and then call all possible opcodes. Verifying if there was evm exception or no. (Undefined opcodes cause exception) + the same must now be done in eof context.